package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "test",
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build",
+    "preview": "vite preview",
+    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+    "format": "prettier --write ."
```

Comment:
Do we have a script to run eslint or to check for prettier issues? is that the same as `check`?

Reply:
![Screenshot 2024-12-04 at 9 33 19 PM](https://github.com/user-attachments/assets/bf023770-b07a-486d-9abc-f3dc0a257041)
yeah pnpm format runs the formatter. check will tell u if u have errors

Reply:
im saying does this `pnpm check` does this run eslint as part of svelte-check or do u still have to run eslint?